### PR TITLE
doc: update diff report generation doc with label examples

### PR DESCRIPTION
**Pull request by strkkk**
Necessary doc update after https://github.com/checkstyle/checkstyle/pull/9199 is merged

Updated section can be viewed here https://github.com/strkkk/contribution/tree/label_info/checkstyle-tester#executing-generation

### CODE REVIEW

File: checkstyle-tester/README.md [hunk 1]
```diff
@@ -118,14 +118,21 @@ Diff Regression projects: {{URI to projects-to-test-on.properties}}
 
 Diff Regression config: {{URI to my_checks.xml}}
 
-If you want to generate [Difference Report with Different Base and Patch Config]
-(./README_MANUAL_EXECUTION.md#difference-report-with-different-base-and-patch-config)
+If you want to generate [Difference Report with Different Base and Patch Config](./README_MANUAL_EXECUTION.md#difference-report-with-different-base-and-patch-config)
```
Comment (strkkk) [inline checkstyle-tester/README.md:121]: link was broken because of whitespace between `]` and `(`

Comment (pbludov) [reply checkstyle-tester/README.md:121]: interesting. mdl should report it.

**Review by pbludov — APPROVED** [review]

File: checkstyle-tester/README.md [hunk 2]
```diff
@@ -118,14 +118,21 @@ Diff Regression projects: {{URI to projects-to-test-on.properties}}
...
 This URI must refer to patch config. The additional URI format should be like this:
 
 Diff Regression patch config: {{URI to patch_config.xml}}
 
+Optional - you can add some label for your report. It can be useful in case you are creating many
+ reports. You need to add special line to PR description:
+
+Report label: here is some label
+
+Everything between "Report label: " and EOL will be taken as a label for report. For example above,
```
Comment (rnveach) [inline checkstyle-tester/README.md:133]: `label for report` => `label for the report`
`For example above` => `For the example above`

Comment (strkkk) [reply checkstyle-tester/README.md]: done

**Pull request merged**